code/modules/atmospherics/environmental/LINDA_fire.dm

```diff
@@ -240,6 +240,13 @@
 		var/mob/living/L = AM
 		L.fire_act(temperature, volume)
 
+// for special backblast hotspots
+/obj/effect/hotspot/backblast/Crossed(atom/movable/AM, oldLoc)
+	..()
+	if(isliving(AM))
+		var/mob/living/L = AM
+		L.adjust_fire_stacks(3) // +3 firestacks on top of the normal +3
+
```

Comment:
Can you explain the logic behind this? I'd like to avoid snowflaking the core atmos code as much as possible

Reply:
It's hotter than normal fire!

Reply:
abandoned the special hotspot